pennylane/transforms/split_non_commuting.py

```diff
@@ -216,6 +304,48 @@ def circuit(x):
     .. details::
         :title: Usage Details
 
+        **Shot distribution**
+
+        With finite-shot measurements, the default behaviour of ``split_non_commuting``
+        will perform ``shots`` executions for each group of commuting terms. With the
```

Comment:
```suggestion
        will use 1 execution with ``shots`` each for each group of commuting terms. With the
```
Current working makes it sound like we are doing "shots" exeuctions, instead of multiple executions with that number of shots.